examples/langchain-ollama/README.md

```diff
@@ -0,0 +1,51 @@
+# LangChain-Ollama Examples
+
+## Install Ollama
+First need to install ollama in local 
+
+visit to [ollama website](https://ollama.com/download)
+
+After download pull lamma 3.1 in local and start 
+```
+ollama run llama3.1
+```
+
+
+
+Two ways to get started:
+
+1. [If you're comfortable with virtualenvs](#if-youre-comfortable-with-virtualenvs)
+2. [If you prefer docker](#if-you-prefer-docker)
+
+## If you're comfortable with virtualenvs
+
+Activate a new virtualenv however you prefer, for example:
+
+```
+python3 -m venv venv
+source venv/bin/activate
+```
+
+install requirements
+
+```
+pip install -r requirements.txt
+```
+
+```
+python 01-math_example.py
+```
+
+## If you prefer docker
+
+```
+docker compose run examples 01-math_example.py
```

Comment:
for me, running ollama outside docker, the script can't reach the host - I think maybe you want to set the env var in compose to use `host.docker.internal` instead of `localhost` ? 

e.g.

```diff
version: "3.7"
services:
  examples:
    build:
      context: .
      dockerfile: Dockerfile
    volumes:
      - ./:/app
    environment:
      - OPENAI_API_BASE=http://host.docker.internal:3090/v1/etc
```

I don't recall correct ports/env var to point to where ollama is running but let me know if that's enough to get you started - if you don't have a docker env let me know and I can help test a fix